modules/rosa-scaling-worker-nodes.adoc

```diff
@@ -0,0 +1,57 @@
+:_module-type: PROCEDURE
+//Specify the system-module-type as either "CONCEPT, PROCEDURE, or REFERENCE"
+
+// Module included in the following assemblies:
+//
+// * assemblies/rosa-managing-worker-nodes.adoc
+
+[id="rosa-scaling-worker-nodes_{context}"]
+= Scaling worker nodes
+
+[role="_abstract"]
+Worker nodes can be scaled manually if you do not want to configure node autoscaling.
+
+. Procedure
+
+. To find the ID of the machine pool you want to scale and verify the current number of replicas, enter the following command to get a list of the machine pools in a cluster. Each cluster has a default machine pool that is created when you create a cluster.
++
+[source,terminal]
+----
+$ rosa list machinepools --cluster=<cluster_name>
+----
++
+.Example output
++
+[source,terminal]
+----
+ID        AUTOSCALING   REPLICAS    INSTANCE TYPE  LABELS    TAINTS   AVAILABILITY ZONES
+default   No            2           m5.xlarge                        us-east-1a
+mp1       No            2           m5.xlarge                        us-east-1a
+----
+
+. To change the number of replicas, or scale, of the machine pool, enter the following command:
```

Comment:
If it's not necessary to include both "replicas" and "scale" here, I'd suggest removing "scale". I know ISG doesn't love parentheses ["number of replicas (scale) of the machine pool"], but the commas here make it hard to tell if the words are synonymous or two different concepts.

Reply:
I miss parenthesis in these instances. But, I have reworded to try to get across the concept that it is the number of replicas that determine the scale. Thanks!